index.html

```diff
@@ -0,0 +1,130 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Elearnig Website 2022, 2023</title>
+    <link rel="stylesheet" href="./style.css" >
+    <script defer src="./script.js"></script>
+    <script src="https://kit.fontawesome.com/10729b3e67.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+  </head>
+  <body>
+      <header>
+        <div class="top-header">
+          <figure class="humburger fa-beat" style="--fa-animation-duration: 3.5s;">
+            <span  class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+           </figure>
+           <a href="#" id="exit" class="nullifier"><i class="fa-solid fa-xmark fa-2xl fa-beat" style="--fa-animation-duration: 2s;"></i></a>
+          <ul>
+            <li><a href="#"><i class="fa-brands fa-facebook-f fa-lg"></i></a></li>
+            <li><a href="#"></a><i class="fa-brands fa-twitter fa-lg"></i></li>
+            <li>English</li>
+            <li>MyPage</li>
+            <li>Layout</li>
+          </ul>
+         </div>
+         <div class="sec-header">
+          <div class="logo">
+            <a href="./index.html"> <img src="./images/logo.png" alt="logo for e learning website" width="100" height="50"></a>
+          </div>
+          <nav class="nav">
+           <ul class="toggle">
+                <li><a href="./about.html" class="">About</a> </li>
+                <li><a href="#" class="">Program</a> </li>
+                <li><a href="#" class="">Courses</a> </li>
+                <li><a href="#" class="start-learning" id="start-learning">Start Learning</a></li>
+             </ul>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <section>
+          <h6>"Hello ! Welcome"</h6>
+          <h1 class="tittle">  E-learning MODERN WAY OF <br> LEARNING 2022, <br>2023 </h1>
+          <div class="invite-wrap">
+            <p>Study your Courses anytime and anywhere</p>
+            <p>Access the study material unlimited times with a quick mode of delivery</p>
+          </div>
+          <h2 class="a-date">22nd Nov 2022 (THU) To <img src="images/infinity.png" id="infinity"></img></h2>
+          <p class="adrasha">@ International Conference on Climate Change (ICCC)</p>
+        </section>
+        <section class="main-program">
+          <h2>Main Courses</h2>
+          <div class="bar-icon"></div>
+          <div class="card-container">
```

Comment:
Great design @Yordinia 🤝

- [ ] Kindly make sure your design replicates the design provided as much as possible. You may make this section close to the design given.
- Your design
![your main-program](https://user-images.githubusercontent.com/63917725/205318662-ca5b2122-1cfd-4596-b20e-700ea4c5f983.PNG)
- Given design
![given main-program](https://user-images.githubusercontent.com/63917725/205318753-6a84c4cd-ffc9-4241-ab27-a6805c91fd99.PNG)